.prettierrc

```diff
@@ -2,12 +2,10 @@
   "printWidth": 80,
   "tabWidth": 2,
   "singleQuote": true,
-  "trailingComma": "es5",
   "bracketSpacing": true,
   "jsxBracketSameLine": false,
```

Comment:
shell we get ride of `jsxBracketSameLine` as well?

Reply:
what about going back to a clean configuration that only requires non-defaults? 

```
{
  "singleQuote": true,
}
```

Or you prefer keeping the options as doc? I see people often going into our source an copying the whole thing, it helps to see the only opinionated thing is the singleQuote.

Reply:
I vote for cleaning it up.

Reply:
👍